### PR TITLE
Fix Docker Permission for Serve release test again

### PR DESCRIPTION
**Pull request by simon-mo**
Somehow missed in the past.
+ Added a piggyback commit to set expiration="never" for long running tests. we will have a separate process of managing that.

### CODE REVIEW

File: python/ray/serve/benchmarks/cluster.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ worker_nodes:
 initialization_commands: []
 setup_commands:
     - sudo apt-get install build-essential libssl-dev git -y
-    - 'rm -r wrk || true && git clone https://github.com/wg/wrk.git wrk && cd wrk && make -j && cp wrk /usr/local/bin'
+    - 'rm -r wrk || true && git clone https://github.com/wg/wrk.git /tmp/wrk && cd /tmp/wrk && make -j && sudo cp wrk /usr/local/bin'
```
Comment (ijrsvt) [inline python/ray/serve/benchmarks/cluster.yaml:30]: shouldn't this be
```
sudo cp /tmp/wrk /usr/loca/bin
```

File: python/ray/serve/benchmarks/single.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ head_node:
 initialization_commands: []
 setup_commands:
     - sudo apt-get install build-essential libssl-dev git -y
-    - 'rm -r wrk || true && git clone https://github.com/wg/wrk.git wrk && cd wrk && make -j && cp wrk /usr/local/bin'
+    - 'rm -r wrk || true && git clone https://github.com/wg/wrk.git /tmp/wrk && cd /tmp/wrk && make -j && sudo cp wrk /usr/local/bin'
```
Comment (ijrsvt) [inline python/ray/serve/benchmarks/single.yaml:25]: Same as before

File: release/long_running_tests/cluster.yaml [hunk 3]
```diff
@@ -15,3 +15,10 @@ auth:
 
 head_node:
     InstanceType: m5.xlarge
+    TagSpecifications:
+        - ResourceType: "instance"
+          Tags:
+            - Key: anyscale-user
+              Value: "release-automation@anyscale.com"
+            - Key: anyscale-custodian
+              Value: "ignore"
```
Comment (ijrsvt) [inline release/long_running_tests/cluster.yaml:24]: Should this be truly unlimited ?

**Review by ijrsvt — APPROVED** [review]
LGTM, a few comments, but all minor.

File: python/ray/serve/benchmarks/cluster.yaml (hunk 1 above)
Comment (simon-mo) [reply python/ray/serve/benchmarks/cluster.yaml:30]: nope we are coping the binary so it's actually `/tmp/wrk/wrk` but we cd there already

File: release/long_running_tests/cluster.yaml (hunk 3 above)
Comment (simon-mo) [reply release/long_running_tests/cluster.yaml:24]: yeah we will have cost monitoring setup. and these are small instances

File: python/ray/serve/benchmarks/cluster.yaml (hunk 1 above)
Comment (ijrsvt) [reply python/ray/serve/benchmarks/cluster.yaml:30]: Ahh okay!

**Pull request merged**